lib/d3-flame-graph/readme_moodle.txt

```diff
@@ -0,0 +1,17 @@
+
+Home page for d3-flamegraph is https://github.com/spiermar/d3-flame-graph.
+
+You will need NPM isntalled on your system to perfrom the follwoing procedure.
+
+Procedure for installing d3-flame-graph
+* Create a suitable directory.
+* Perform the following commands.
+
+$ git clone https://github.com/spiermar/d3-flame-graph.git
+$ cd d3-flame-graph
+$ npm install
+
+* Copy contents of 'dist' directory to lib/d3-flamegraph in plugin.
+* Copy LICENSE to lib/d3-flamegraph in plugin.
```

Comment:
I wonder if the whole thing could be commands we can use to do a quick update.
```suggestion
* Copy contents of 'dist' and license directory to lib/d3-flamegraph in plugin.
cp -r somewhere/dist somewhere/LICENSE -t lib/d3-flame-graph
```

Reply:
Also in your example the target folder has a different name from the one you've provide in this commit

lib/d3-flamegraph => lib/d3-flame-graph

Reply:
also worth having a step to fully remove the dir contents first so we aren't merging files across versions